rubocop/rubocop_rails.yml

```diff
@@ -3,11 +3,34 @@ inherit_from: rubocop.yml
 AllCops:
   Exclude:
     - db/schema.rb
+    - 'bin/**/*'
+    - 'config/**/*'
     - 'vendor/**/*'
 
 Rails:
   Enabled: true
 
+Metrics/BlockLength:
+  Exclude:
+    - 'db/seeds.rb'
+    - 'spec/**/*'
```

Comment:
This is already disabled in our [base](https://github.com/TheGnarCo/gnar-style/blob/86e5b8c14efa8d4601c532f062740850254ce365/rubocop/rubocop.yml#L39) cop file. This should be unnecessary, and if you're seeing notifications for violating this cop in spec files right now, that points to a different bug or configuration setting that should be addressed differently from specifying it here.